azure-pipelines.yml

```diff
@@ -498,7 +502,34 @@ stages:
       displayName: Call repox
       steps:
       - checkout: none
-
+      - powershell: |
+          $isPr=$env:BUILD_REASON -eq 'PullRequest'
+          $repoPrefix = 'sonarsource-public'
+          $it = Get-Date -Format s
+          if ($isPr)
+          {
+            $targetRepo = $repoPrefix + '-dev'
+            $repoxStatus = 'it-passed-pr'
+          } 
+          else
+          {
+            $targetRepo = $repoPrefix + '-builds'
+            $repoxStatus = 'it-passed'
+          }
+          Write-Host "##vso[task.setvariable variable=targetRepo;]$targetRepo"
+          Write-Host "##vso[task.setvariable variable=repoxStatus;]$repoxStatus"
+          Write-Host "##vso[task.setvariable variable=it;]$it"
```

Comment:
This is the converted python script. I think we can also inline the calculations into the jfrog task itself, but I haven't tried this just yet.